src/css/Header.scss

```diff
@@ -0,0 +1,42 @@
+.headerWrap {
+  background-color: #C5E9FF;
+  width: 100%;
+  height: 117px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-shadow: 4px 4px 4px 4px rgba(0, 0, 0, 0.25);
+
+
+  .header {
+    width: 1000px;
+    display: flex;
+    align-items: center;
+
+    svg {
+      margin: 0 30px 0 0;
+    }
+
+    p {
+      font-family: 'Roboto', sans-serif;
+      font-style: normal;
+      font-weight: 300;
+      font-size: 36px;
+      line-height: 42px;
+      margin: 0 30px;
+    }
+
+    input {
+      font-family: 'Roboto', sans-serif;
+      font-style: normal;
+      font-weight: 300;
+      font-size: 18px;
+      padding: 10px 20px;
+      background: #FFFFFF;
+      border: 1px solid rgba(0, 0, 0, 0.2);
+      box-sizing: border-box;
+      border-radius: 4px;
+    }
+  }
+  
+}
```

Comment:
And here please check all the styles for the design. And nowhere should be the exact values